tests/.nftignore

```diff
@@ -1,6 +1,12 @@
 .DS_Store
+Ancient_DNA/pydamage/analyze/*/*.csv
+Ancient_DNA/variant_calling/*/*.vcf.gz
+Annotation/Prokka/**/*.{log,err,gbk,sqn}
+Annotation/Prokka/**/*.tmp.*
 Assembly/MEGAHIT/*.log
 Assembly/MEGAHIT/QC/*/*.bowtie2.log
+Assembly/MEGAHIT/QC/*/QUAST/*.{pdf,html,log}
```

Comment:
To verify, there is no 'nice' 'completed successful' entry in the `log` file? Other two are fine